main_app/templates/base.html

```diff
@@ -1,61 +1,73 @@
 {% load static %}
-
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>I M S</title>
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
-  <link rel="stylesheet" type="text/css" href="{% static 'css/style.css' %}">
-</head>
-
-<body>
-  <header class="navbar-fixed">
-    <nav>
-      <div class="nav-wrapper">
-        <ul>
-          <li><a href="{% url 'home' %}" class="left brand-logo">&nbsp;&nbsp;Inventory Management System</a></li>
-        </ul>
-        <ul class="right">
-
-          <li ><a href="#" class="dropdown">Features</a></li>
-          <div class="dropdown-content">
-            <a href="{% url 'about' %}">About</a>
-            <a href="#">Contact</a>
-          </div>
-          <li><a href="#">Pricing</a></li>
-
-          {% if user.is_authenticated %}
-
-          <li><a href="#" class="dropdown">Support</a></li>
-          <div class="dropdown-content">
-            <a href="#">Knowledge Base</a>
-            <a href="#">FAQs</a>
-          </div>
-          <li><a href="{% url 'item_create' %}">Add an Item</a></li>
-          <li><a href="{% url 'item' %}">View My Inventory</a></li>
-          <li class="nav-item">
-            <a class="nav-link" href="{% url 'logout' %}">Logout</a>
-          </li>
-
-          {% endif %}
-
-
-        </ul>
+  <head>
+    <title>Inventory Management System</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+    <link href="{% static 'css/bootstrap.min.css' %}" rel="stylesheet">
+  </head>
+  <body class="d-flex flex-column min-vh-100">
+    <nav class="navbar navbar-expand-lg navbar-dark bg-primary">
+      <div class="container-fluid">
+        <a class="navbar-brand" href="{% url 'home' %}">
+          <i class="fa-solid fa-synagogue fa-lg"></i> &nbsp;&nbsp;StockSense
+        </a>
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarColor01" aria-controls="navbarColor01" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarColor01">
+          <ul class="navbar-nav me-auto">
+            <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" data-bs-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">Features</a>
+                <div class="dropdown-menu">
+                  <a class="dropdown-item" href="{% url 'about' %}">About</a>
+                  <a class="dropdown-item" href="#">Contact</a>
+                  <a class="dropdown-item" href="#">Pricing</a>
+                </div>
+              </li>
+              {% if user.is_authenticated %}
```

Comment:
Having everything except the "Features" dropdown menu on the nav bar hidden for users who aren't logged on seems a little lonely. Thoughts?

Reply:
Yeah I tend to agree, what features were we hoping to have here and are they clickable links?